libs/hdf-converters/src/sonarqube-mapper.ts

```diff
@@ -255,9 +278,18 @@ function createSonarqubeMappings(
 
 export class SonarQubeMapper extends BaseConverter {
   projectName = '';
-  constructor(issuesJSON: IssueData, projectName: string) {
+  branchName = '';
+  pullRequestID = '';
+  constructor(
+    issuesJSON: IssueData,
+    projectName: string,
+    branchName?: string,
+    pullRequestID?: string
+  ) {
     super(issuesJSON as Record<string, any>);
-    this.setMappings(createSonarqubeMappings(projectName));
+    this.setMappings(createSonarqubeMappings(
+      projectName, branchName, pullRequestID)
+    );
   }
 
   setMappings(
```

Comment:
Can you remove this function?  We've been removing it when it's unnecessary.  All it's doing here is calling the one in the BaseConverter, at which point we might as well use the BaseConverter's implementation directly.

https://github.com/mitre/heimdall2/blob/b5bac50274b935ee349a3018a8d328bcbb24e2a8/libs/hdf-converters/src/base-converter.ts#L147=